### DISCUSSION TITLE
The color legend of 2 charts combining together - Any workaround for this ?

### DISCUSSION THREAD

**Discussion by nareshkumarhari** [vega/altair#2729] [General]
I have two rows of charts with different attributes used for colour. But the legend is combined. Not sure why and how to overcome this.

Zipcode is used as color encoding for first chart and price range is used for chart on Row 2.

![image](https://user-images.githubusercontent.com/68188456/204374960-948b6ca8-b24b-4552-92c7-64c1e429ec6f.png)

@jakevdp - Can you please help me with this

**Comment by jakevdp** [comment]
You can control this using the tools described at [Scale and Guide Resolution](https://altair-viz.github.io/user_guide/scale_resolve.html#scale-and-guide-resolution)

**Reply by nareshkumarhari** [reply]
@jakevdp - You are super fast. Thank you for your prompt response. I will check now and will confirm shortly.

**Reply by nareshkumarhari** [reply]
@jakevdp  - Thank you so much for your help. Now it's working as expected. You are a superstar.
